extension/ts/main/extensions/util/onLoggedIn.ts

```diff
@@ -0,0 +1,35 @@
+import {invokeWorker} from "../../../common/workers/invoker";
+import {WorkerKind} from "../../../common/workers/types";
+import {createIconButton} from "../../ui/button";
+import {isAuthorized} from "../author/util/isAuthorized";
+
+const authorize = (interactive: boolean) => invokeWorker(WorkerKind.Authorize, interactive).catch(() => "");
+
+function injectButton(button: HTMLTableCellElement, container: HTMLElement) {
```

Comment:
what happened to our beautiful arrow functions :(

Reply:
oop